README.md

```diff
@@ -114,5 +114,12 @@ server.start();
 Pothos development supported by [sponsors](https://github.com/sponsors/hayes) from these generous
 people:
 
+## Sponsors
+
+Pothos development supported by [sponsorships](https://github.com/sponsors/hayes) from these
+generous people and organizations:
+
+- [![The Guild](https://pothos-graphql.dev/assets/the-guild-logo.svg)](https://www.the-guild.dev/)
 - [@kidqueb](https://github.com/kidqueb)
 - [@homoky](https://github.com/homoky)
+- [@huv1k](https://github.com/https://github.com/huv1k)
```

Comment:
Here is t wice GitHub link 😅

Reply:
Whoops, fixed

Reply:
Also, thank you for the sponsorship!

Reply:
I thank you for great framework 🙌  https://mobile.twitter.com/huv1k/status/1514500281823862784